src/assets/labContent.js

```diff
@@ -150,27 +159,19 @@ export default [
       subheader: "How deep do the effect of advertisements run in society itself?",
       bodyText: [
         {
-          body: `Here is a graph displaying the gender distribution of computer scientists from the 1960s to now.`
+          body: <text>Here is a graph displaying the <b>gender distribution of computer scientists from the 1960s to now.</b></text>
         },
         {
           body: <PlaceholderInteractive />
         },
         {
-          body: `Did you notice the sudden drop in females? This was caused by advertisements geared towards the male-dominated nerd culture of the 80s.`
-        },
-        {
-          body: `At the time, "nerdy" interests such as video games and computers were commonly viewed as interests that women did not want to participate in. Advertisements transferred that misconception to its viewers, causing women to feel alienated from those activities.`
-        },
-        {
-          body: `The results can be seen in the graph: advertisements had turned the misconception into an unfortunate reality.`, 
-          bold: true
+          body: `Did you notice the sudden drop in females?`
         },
         {
-          body: `Despite numerous efforts to lower the gender disparity in tech today, the current distribution has a long way to go until we can erase the sever impact of past advertisements.`
+          body: <text>In the 60s when computers first emerged, women made up most of the computing field. <u>However, in the 70s, personal computers were advertised almost exclusively to men,</u> influencing families to buy computers for boys than girls. While the percentage of women in CS rose in the 80s thanks to the boom of nerd culture, since then it has been in decline.</text>
```

Comment:
Same text tag comment as above, I will stop adding text comments in my review but please be sure to adjust the rest!